AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_materialproviders/smithygenerated/aws_cryptography_keystore/client.py

```diff
@@ -0,0 +1,462 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+import asyncio
+from asyncio import sleep
+from copy import deepcopy
+from software_amazon_cryptography_keystore_internaldafny_types import IKeyStoreClient
```

Comment:
Do you have a separate branch with `.` replaced with `_` in the dafny source code?

Reply:
No, it's not a separate branch.
I'm keeping any `.`/`_` things out of scope until I get to per-module outer-module names.

Reply:
So it's a local change and you are using that to generate this code?

Reply:
Oh, I guess there actually is a separate branch, which is just the "authoritative" branch for these changes: https://github.com/aws/aws-cryptographic-material-providers-library/tree/lucmcdon/python-mpl

Ex. I've committed changes like this to the Dafny source code: https://github.com/aws/aws-cryptographic-material-providers-library/blob/lucmcdon/python-mpl/AwsCryptographyPrimitives/src/Index.dfy